backend/.env.example

```
@@ -79,3 +79,18 @@ OPENSEARCH_USERNAME=admin
 OPENSEARCH_PASSWORD=admin
 OPENSEARCH_SSL_REJECT_UNAUTHORIZED=false
 OPENSEARCH_REFRESH=false
+
+# Tazama Authentication (RSA Public Key)
+TAZAMA_AUTH_URL=http://10.10.80.30:3020/v1/auth
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The TAZAMA_AUTH_URL key is duplicated

(DuplicatedKey)